utils/python/tests/scripts_regression_tests.py

```diff
@@ -170,6 +170,20 @@ def setup_proxy():
     return False
 
 ###############################################################################
+class N_TestUnitTest(unittest.TestCase):
```

Comment:
It would be nice to add the cime unit tests, but not until they are working or failing tests are removed.